### DISCUSSION TITLE
第4节课笔记(2班)

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/Tutorial#244] [笔记]
## 2班第四节课笔记交这里哟~
* 链接形式提交
* 也记得在信息收集文档对应作业处填写链接

**Comment by songyuc** [comment]
[【书生·浦语大模型实战营04】《(4)XTuner 大模型单卡低成本微调实战》学习笔记](https://blog.csdn.net/songyuc/article/details/135512470)

**Comment by xrxu** [comment]
第四课笔记: https://github.com/xrxu/Docu/blob/master/xtuner/xtuner.md

**Comment by DarthThomas** [comment]
[第4课笔记](https://share-tianhe.notion.site/9f8fed156816461eb327139a8d900c3e?pvs=4)

**Comment by pranerd** [comment]
[第四节课笔记](https://blog.csdn.net/gaxzj/article/details/135586441?spm=1001.2014.3001.5502)

**Comment by xulinui** [comment]
https://h90ag9106t.feishu.cn/docx/YjQPdpEJpohTe8xF2UucCguanbe

**Comment by qihong2019** [comment]
[第4节 XTuner 大模型单卡低成本微调实战](https://www.processon.com/view/link/65a3e55fecbb0c212a2ef03c)
![第4节 XTuner 大模型单卡低成本微调实战](https://github.com/InternLM/tutorial/assets/47970679/7ddcac07-7f01-41c8-a97b-731cbebb0fda)

**Comment by CoderJackZhu** [comment]
[第四节笔记](https://zhuanlan.zhihu.com/p/677771386)

**Comment by ljqiff** [comment]
[第四次笔记Xtuner](https://blog.csdn.net/qq_25820969/article/details/135591079?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135591079%22%2C%22source%22%3A%22qq_25820969%22%7D)

**Comment by jinxianwei** [comment]
[第四节课笔记](https://blog.csdn.net/jinxianwei1999/article/details/135608194?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135608194%22%2C%22source%22%3A%22jinxianwei1999%22%7D)

**Comment by yangff0016** [comment]
[第四节课笔记](https://blog.csdn.net/weixin_42837310/article/details/135632985)

**Comment by ZixinxinWang** [comment]
https://blog.csdn.net/Wzxdecsdn/article/details/135581912

**Comment by RyanWu31** [comment]
https://github.com/RyanWu31/tutorial_notebook/blob/main/README.md

**Comment by Mr-Vincenyang** [comment]
[笔记](https://zhuanlan.zhihu.com/p/678719352)

**Comment by YanxingLiu** [comment]
第四节课笔记 https://s5eixc6gba.feishu.cn/docx/UdgbdrKiToXG7Ax8kPOc5QNdnPh?from=from_copylink

**Comment by zyy0831** [comment]
第四节课笔记 https://www.yuque.com/yuanyang-xb8hd/eeag2y/kpnupg6hblsq34gz

**Comment by PrinceRay7** [comment]
第四节课笔记
https://blog.csdn.net/Prince_Ray/article/details/135832619?spm=1001.2014.3001.5501

**Comment by kishiokon** [comment]
[XTuner 大模型单卡低成本微调实战](https://www.bilibili.com/read/cv30144823/?jump_opus=1)
